charts/pega/README.md

```diff
@@ -529,6 +529,21 @@ tier:
       serviceAccountName: MY_SERVICE_ACCOUNT_NAME
 ```
 
+### Sidecar Containers
+
+If the pod needs to run with one or more [sidecar containers](https://kubernetes.io/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers), you can specify custom 'sidecarContainers' for your deployment tier.
+
+Example:
+
+```yaml
+tier:
+  - name: my-tier
+    custom:
+      sidecarContainers:
+        - name: SIDECAR_NAME
+          image: SIDECAR_IMAGE_URL
```

Comment:
IDK if it's worth calling out in this README, since if you're using a sidecar container you probably know or have some guide for the required `spec`... but a container needs some `command` and the `resources` requests. From first part of comment and sake of keeping README concise, I totally understand if it's intentional to leave this as is

Reply:
Updated with more indication that you do need more than simply the name and image. I don't think the command is always necessary, but resources should be there at least, and there's likely to be more. Using `...` to denote extra stuff, other examples here use that notation.

Reply:
I vote for leaving them out since they're not required and might make things more confusing.  Although maybe we could have some sort of phrasing that indicates you could expand upon the name and image to include more fields describing the container.